storage/src/adb/current.rs

```diff
@@ -90,16 +110,18 @@ impl<
         const N: usize,
     > Current<E, K, V, H, T, N>
 {
-    // A compile-time assertion that the chunk size is the expected multiple of digest size.
-    const _MULTIPLE: usize = 2; // 2 yields the smallest possible proof sizes
+    // A compile-time assertion that the chunk size is some multiple of digest size. A multiple of 1 is optimal with
+    // respect to proof size, but a higher multiple allows for a smaller (RAM resident) merkle tree over the structure.
+    const _MULTIPLE: usize = 1;
     const _CHUNK_SIZE_ASSERT: () = assert!(
         N == Self::_MULTIPLE * H::Digest::SIZE,
```

Comment:
If we want to allow any multiple, would it be better to just do `N % H::Digest::SIZE == 0 && N > 0`?

The current assertion requires them to just be equal, no?

Reply:
yeah, seems I updated the comment without updating the code. fixed.